README.md

```diff
@@ -68,20 +73,35 @@ To use the default configuration as a template you can use:
     blugon --printconfig > ~/.config/blugon/config
 
 ## Dependencies
+
 - `python`
 - `libx11`
 - `libxrandr`
+
 ### optional:
+
 - `xorg-xgamma` as backend
 
 ## Install
+
 ### ArchLinux
+
 There is an [AUR-Package](https://aur.archlinux.org/packages/blugon).
 
+### Debian
+
+    $ git clone https://github.com/jumper149/blugon.git
+    $ sudo apt install gcc make python3 libx11-dev libxrandr-dev
+    $ cd blugon/
+    $ make PREFIX=/usr/local
+    $ sudo make install PREFIX=/usr/local
+
```

Comment:
This is pretty much the same as the "from source"-section. Listing the dependencies might be a good idea still. Knowing how to use git will be necessary and not explicitly explained in this readme.

Edit: This should then go into the "Dependencies"-section.

Reply:
You are right that my change introduces unnecessary redundancy.

There is one aspect I want to mention explicitly: On Debian systems, when manually installing software, you write to `usr/local/`, as `/usr/` is more or less reserved for installing via `apt`.

Reply:
> There is one aspect I want to mention explicitly: On Debian systems, when manually installing software, you write to `usr/local/`, as `/usr/` is more or less reserved for installing via `apt`.

This would still be the same as in the "from source"-section. I can see that helping someone, but I would still like to put the focus on a concise readme. The extra subsubsections under "Install" should refer to proper packages only.

Regarding different paths for different distributions:
These should be explained in something like an OS-manual and not here imo.

Reply:
Alright :-)